internal/vsp/vsp.go

```diff
@@ -138,6 +138,15 @@ func (c *Client) ProcessUnprocessedTickets(ctx context.Context, policy Policy) {
 		if err == nil {
 			return nil
 		}
+		confirmed, err := c.Wallet.IsVSPTicketConfirmed(ctx, hash)
+		if err != nil && !errors.Is(err, errors.NotExist) {
+			// Should we just log and return nil here?
+			return err
```

Comment:
Probably log either here or this needs to be refactored so that the error returned in the `c.ForUnspentUnexpiredTickets` above isn't simply ignored.